projects/jsoncpp/json.proto

```diff
@@ -10,8 +10,9 @@ syntax = "proto2";
 package json_proto;
 
 message JsonObject {
-  required string name = 1;
-  required JsonValue value = 2;
+  required int32 settings = 1;
```

Comment:
@dannyallover 

By placing settings here, this ends up causing trouble with recursion on line 25; that is, if would create

```
JsonObject {
  settings = 4,
  name = 'outer'
  value = JsonValue {
    JsonObject {
      settings = 1
      name = 'inner'
      value = StringValue {
        value = "Foo"
      }
    }
  }
}
```

Instead, you probably want something like:

```protobuf
message ParseAPI {
  required int32 settings = 1;
  required JsonValue value = 2;  // Or JsonObject, if the parse API expects valid objects
}
```

This makes sure your settings are only set for the outermost API call, and your subsequent recursions don't pass the settings.

While you're here, it *might* make sense to extend this fuzzer to create properly complex objects:

```protobuf
message JsonField {
  required string name = 1;
  required JsonValue value = 2;
}

message JsonObject {
  repeated JsonField field = 1;
}
```

This will create objects with >1 field, which can handle things like what happens if a field starts as a string value and then becomes a number-value during parsing.